ipv-stub/src/endpoints/ipv-userinfo.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  APIGatewayProxyEvent,
+  APIGatewayProxyResult,
+  Handler,
+} from "aws-lambda";
+import {
+  handleErrors,
+  methodNotAllowedError,
+  successfulJsonResult,
+} from "../helper/result-helper";
+
+export const handler: Handler = async (
+  event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> => {
+  return handleErrors(async () => {
+    switch (event.httpMethod) {
+      case "GET":
+        return await handle(event);
+      case "POST":
```

Comment:
Do we want to support both methods here?

Reply:
Due to a lack of internal specification atm, I've defaulted to yes for now due to the OIDC spec saying UserInfo MUST support both GET and POST. We could remove this later.

https://openid.net/specs/openid-connect-core-1_0.html#UserInfo

> The UserInfo Endpoint MUST support the use of the HTTP GET and HTTP POST methods defined in RFC 7231 [RFC7231].

Reply:
Since doing this, we've deduced this is not a traditional OIDC userinfo endpoint, so removing POST